lib/src/modules/task.cc

```diff
@@ -426,9 +438,110 @@ static fs::path getCachedTaskFile(const fs::path& task_cache_dir,
     }
 }
 
+// build a spool dir for the supporting library files requested by a multifile task
+static fs::path createInstallDir(const fs::path& spool_dir, const std::set<std::string>& download_set) {
+    auto install_dir = spool_dir / fs::unique_path("temp_task_%%%%-%%%%-%%%%-%%%%");
+    fs::create_directories(install_dir);
+    fs::last_write_time(install_dir, time(nullptr));
```

Comment:
This should be unnecessary because we're always creating a new directory. It's only done elsewhere when we reuse an existing directory.